guide/14-deep-learning/faster-rcnn-object-detector.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>Using a sliding window approach, a small network is <s>used over</s> <em>overlaid onto </em>the feature map. At each spatial window, there are multiple anchor boxes that are of predetermined scales and aspect ratios. This enables the model to detect objects of a wide range of scales and aspect ratios in the same image. Usually, three different scales and three different aspect ratios are used, resulting in nine anchor boxes at each spatial location<s>, which</s> <em>that </em>denote<s>s</s> the maximum region proposals at that spatial location. The small network then feeds into fully connected layers with two heads - one for the "objectness" score and the other for the bounding box coordinates of the region proposals. Figure 5 demonstrates the sliding window at a spatial location, the corresponding anchor boxes, and how <s>it</s> <em>they </em>feed<s>s</s> into the subsequent layers [1]. Note that these layers, though similar to the last few layers of the Fast R-CNN object detector, are not the same, nor do they share the <em>same </em>weights. The RPN classifies the regions in a class agnostic manner, as it is tasked with only finding the regions <s>which</s> <em>that </em>contain the objects. </p><p>In this guide, we have tried to present an intuition to understand the Faster R-CNN model. To dive into more details about the model and it's architecture, please read the original papers mentioned in <em>the </em>references below. You can <em>also </em>reference this <a href="https://towardsdatascience.com/faster-r-cnn-for-object-detection-a-technical-summary-474c5b857b46" target="_blank">blog</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>